### PR TITLE
feat(module:slider): support nzTooltipVisible

### LINKED ISSUE #2373: slider: minimum value should always of zero position in range mode
## 重现

**Angular**

https://stackblitz.com/edit/angular-n2vcdt?file=src/app/app.component.ts

gif:

ng

**React**

https://codepen.io/cipchk/pen/wYZazP?editors=0010

gif:

react

### PR DESCRIPTION
**Pull request by wendzhue**
close #2373

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: components/slider/nz-slider-marks.component.html [hunk 1]
```diff
@@ -1,3 +1,10 @@
-<div [class]="nzClassName">
-  <span *ngFor="let attr of attrs; trackBy: trackById" [ngClass]="attr.classes" [ngStyle]="attr.style" [innerHTML]="attr.label"></span>
+<div class="ant-slider-mark">
+  <span
+    class="ant-slider-mark-text"
+    *ngFor="let attr of marks; trackBy: trackById"
+    [class.ant-slider-mark-active]="attr.active"
+    [ngClass]="attr.classes"
```
Comment (HsuanXyz) [inline components/slider/nz-slider-marks.component.html]: `DisplayedMark` 里面不包含 `classes` 属性，组件其他地方也没看见。

Comment (wendzhue) [reply components/slider/nz-slider-marks.component.html]: I'll fix that.

File: components/slider/nz-slider-handle.component.ts [hunk 2]
```diff
@@ -1,78 +1,128 @@
...
+    if (nzActive) {
+      if (nzActive.currentValue) {
+        this.toggleTooltip(true);
       } else {
-        this.tooltip.hide();
+        this.toggleTooltip(false);
       }
     }
+    if (nzTooltipVisible && !nzTooltipVisible.isFirstChange() && this.tooltip) {
+      this.tooltip.show();
```
Comment (HsuanXyz) [inline components/slider/nz-slider-handle.component.ts]: `nzTooltipVisible.previousValue === 'never'` 时 `this.tooltip` 还不存在，需要异步调用 `this.tooltip.show()` 才会生效。

Comment (wendzhue) [reply components/slider/nz-slider-handle.component.ts]: 如果是 never 的话, 就不需要调用 this.tooltip.show(), 不生效是对的.

**Comment by vthinkxie** [comment]
缺少拖拽时的border样式

**Pull request merged**